resources/services/lightsail/gen.hcl

```diff
@@ -5,23 +5,95 @@ add_generate     = true
 description_modifier "remove_read_only" {
   words = ["  This member is required."]
 }
+#
+#resource "aws" "lightsail" "instances" {
```

Comment:
Why is this being commented out?

Reply:
Ah okay, yeah makes sense. With the new version you should be able to generate without problems :)